### PR TITLE
Feature/rm 2495 add arm64

### PR DESCRIPTION
**Pull request by lzecca78**
Hi everyone 👋 ! 
we've introduced in the goreleaser the arm64 arch for furyagent as well.
Enjoy!

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@ steps:
         - push
 
   - name: build-release-homebrew
-    image: goreleaser/goreleaser:v1.4.1
+    image: goreleaser/export GITHUB_USER=SIGHUP-C-3PO
```
Comment (lnovara) [inline .drone.yml:84]: I guess this is a typo.

**Review by lnovara — APPROVED** [review]
One minor comment. Beside that, LGTM.

**Comment by lnovara** [comment]
Are the changes to the Go code needed to support amd64?

**Comment by nutellinoit** [comment]
> Are the changes to the Go code needed to support amd64?

I think that the changes on go are from the PR #34 , not merged yet

**Comment by omissis** [comment]
I changed the base from master to the pr #34 to show only changes related to this branch

**Review by omissis — APPROVED** [review]

**Pull request merged**